docker/healthcheck.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+database_dir=/usr/share/GeoIP
+if ! [ -z "$GEOIPUPDATE_DB_DIR" ]; then
+  database_dir=$GEOIPUPDATE_DB_DIR
+fi
+
+# The health check is done by checking if the database directory is modified within the
+# update period minus 1 minute. The 1 minute is a threshold for allowing slower starts.
+# Without the LockFile in the database directory, this check is not going to be working
+# since database files are not going to be modified when there are no updates.
+cutoff_date=$(($(date +%s) - $GEOIPUPDATE_FREQUENCY * 60 * 61 ))
+modified_at=$(find $datbase_dir -type f -exec stat -c "%Y" {} + | sort -nr | head -n 1)
```

Comment:
I think we could edit the `find` to only look for the database files with a name convention like `-name  '*.mmdb'`. This way it will be a lot more reliable.

Reply:
If you only look for the database files, you aren't considering the lock file.

Reply:
Yes. I don't think the purpose of the health check is to check the lock file as that can be set to a different location out of the database directory as well. I could be missing any cases where not checking the modification date of the lock file could cause a problem. Do you have anything in mind ? Thanks

Reply:
The purpose of the health check is to make sure the service is functioning correctly. This includes ensuring that it is checking for updated databases on the defined schedule. Since the databases aren't necessarily updated every time the service checks, the lock file was (or it appears that it was intended to be) acting as a proxy for indicating that the service executed on the schedule as expected. With this in mind, I'd suggest that rather than looking for the most recently updated file in some directory, the health check script should be updated to check the lock file explicitly, regardless of its location, so that `HEALTHY` == `I checked for updates`.

Reply:
> Fair enough. I just didn't want to assume anything, so I thought a comprehensive response would remove some uncertainty and ambiguity.

Thank you! I appreciate your input.

I wonder if we could go with another approach and store the individual times that the databases in the config are updated/checked. And, make the health check use that file.

Reply:
I'm sure those suggestions could be fleshed out to provide a much more robust health check, and I'm happy to see improvements to the product. However, if I could be a little self-serving for a moment 😅, I would prefer to see this change just use the lock file as originally intended so that [my open issue](https://github.com/maxmind/geoipupdate/issues/225) can be resolved. From a more objective perspective, though, I would still make the same suggestion to my own team if this was one of our services.